lib/internal/errors.js

```diff
@@ -54,7 +61,6 @@ function message(key, args) {
 // Utility function for registering the error codes. Only used here. Exported
 // *only* to allow for testing.
 function E(sym, val) {
-  assert(messages.has(sym) === false, `Error symbol: ${sym} was already used.`);
```

Comment:
Why did you remove this check? (The test for it is removed in the second commit btw)